pkgs/os-specific/linux/nixos-rebuild/nixos-rebuild.sh

```diff
@@ -60,6 +61,9 @@ while [ "$#" -gt 0 ]; do
         upgrade=1
         upgrade_all=1
         ;;
+      --use-substitutes)
```

Comment:
```suggestion
      -s|--use-substitutes)
```

For consistency with nix-copy-closure.

Reply:
I left it off because I thought another already took it, guess not :)